includes/admin.form.inc

```diff
@@ -454,14 +488,13 @@ function islandora_scholar_admin_xpath_form($form, &$form_state) {
     '#type' => 'textfield',
     '#title' => t('Demo Object'),
     '#field_suffix' => (!empty($form_state['values']['islandora_scholar_xpaths_demo']) && (islandora_object_access(ISLANDORA_VIEW_OBJECTS, islandora_object_load($form_state['values']['islandora_scholar_xpaths_demo'])) == 1)) ? islandora_scholar_xpaths_demo_prefix_callback($form_state['values']['islandora_scholar_xpaths_demo']) : t("Modify to refresh value (PID for demo object is required)."),
-    '#description' => t('PID to check these values against. "@option"', array('@option' => ((!empty($form_state['values']['islandora_scholar_xpaths_demo'])) && (islandora_object_access(ISLANDORA_VIEW_OBJECTS, islandora_object_load($form_state['values']['islandora_scholar_xpaths_demo'])) == 1) ? $form_state['values']['islandora_scholar_xpaths_demo'] : ""))),
+    '#description' => t('PID to check these values against. @option', array('@option' => ((!empty($form_state['values']['islandora_scholar_xpaths_demo'])) && (islandora_object_access(ISLANDORA_VIEW_OBJECTS, islandora_object_load($form_state['values']['islandora_scholar_xpaths_demo'])) == 1) ? $form_state['values']['islandora_scholar_xpaths_demo'] : ''))),
```

Comment:
In view of [my earlier comment](https://github.com/Islandora/islandora_scholar/pull/299#issuecomment-413333397), how about `... t('PID to check these values against: @pid.', array('@pid' => ...`?